R/process_observers.R

```diff
@@ -362,8 +385,6 @@ convert_based_on_mini_table <- function(observer_table, mini_observer_table) {
     left_join(mini_observer_table, by = c("obs3" = "input_name")) %>%
     mutate(obs3 = .data$output_name) %>%
     dplyr::select(-.data$output_name)
-
-  return(observer_table)
```

Comment:
Here and elsewhere the return(x) is removed but since x is assigned to something, I don’t think anything will be returned.